Packs/VaronisSaaS/ReleaseNotes/1_0_4.md

```diff
@@ -0,0 +1,7 @@
+#### Integrations
+##### Varonis SaaS
+- Added varonis-integration header to all of the API calls
+- Multi-value parameters for commands now have pipe separator (|) instead of comma.
+- Varonis SaaS API v9 support.
+- Fixed bug with Close Reason enum mismatch.
+- Updated the Docker image to: *demisto/python3:3.10.13.89009*.
```

Comment:
```suggestion
- Added support for varonis-integration header to all API calls.
- Improved implementation of multi-value parameters for commands, now using pipe separator (|) instead of comma.
- Added support for Varonis SaaS API v9.
- Fixed an issue with predefined values mismatch for the **Close Reason** argument.
- Updated the Docker image to: *demisto/python3:3.10.13.89009*.
```

Reply:
Also you need to add Breaking changes note
You can run this SDK command:
`demisto-sdk update-release-notes -i Packs/<Pack Name> -u revision -bc`
This command will generate a corresponding configuration **JSON file** to the new release notes with `breakingChangesNotes` fields. please add a note with the BC changes you made:  the command args that was removed and the changed in the context.

Reply:
Hi, @jbabazadeh 
I have some issues with this running this command. It runs more than 3h without any result.
Below there are logs from it after 2 runs.

```
[2024-03-19T11:29:01] - [MainThread] - [INFO] - update_rn_manager.py:64 - Starting to update release notes.
[2024-03-19T14:59:47] - [MainThread] - [INFO] - __main__.py:2534 - An error occurred while updating the release notes: New release notes file already found for VaronisSaaS. Please update manually or run `demisto-sdk update-release-notes -i VaronisSaaS` without specifying the update_type.
[2024-03-20T00:37:34] - [MainThread] - [INFO] - logger.py:466 - Log file location: /home/vova/.demisto-sdk/logs/demisto_sdk_debug.log
[2024-03-20T00:37:34] - [MainThread] - [INFO] - update_rn_manager.py:64 - Starting to update release notes.
[2024-03-20T03:47:09] - [MainThread] - [INFO] - __main__.py:2534 - An error occurred while updating the release notes: yml file returned is not of type dict
```

Is it possible to pass validation without running this command? I tried to edit ReleaseNotes/1_0_4.md and 1_0_4.json manually, but it doens't help...

Reply:
The release notes look fine to me at this point. The [failing validations](https://app.circleci.com/pipelines/github/demisto/content/352236/workflows/0e25dcfa-6142-41b8-8a1c-1bbde9293f6e/jobs/743854/parallel-runs/0/steps/0-119?invite=true#step-119-52148_68) are indicating that you've modified the output of the command and it breaks backward compatibility. However, since you've created the breaking changes version files, we should be OK. It just means that we'll need to force merge once the review is over and approved.

Reply:
Thanks for assistance with this!